curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/6464a6d98c43acecb8a7867b.md

```diff
@@ -26,7 +26,7 @@ assert.property(
 )
 ```
 
-You should set the value of the `msg` property to `decimalToBinary(1) returns "1" (base case) and gives that value to the stack below. Then it pops off the stack.`
+You should set the value of the `msg` property to ``` `decimalToBinary(1) returns "1" (base case) and gives that value to the stack below. Then it pops off the stack.` ```
```

Comment:
```suggestion
You should set the value of the `msg` property to `"decimalToBinary(1) returns '1' (base case) and gives that value to the stack below. Then it pops off the stack."`
```

Reply:
There are triple backticks already, I've remove the single backticks around double quotes